cli-plugins/manager/manager.go

```diff
@@ -0,0 +1,190 @@
+package manager
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+	"syscall"
+
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+// ErrPluginNotFound is the error returned when a plugin could not be found.
+type ErrPluginNotFound string
```

Comment:
I think we should expose a function `IsErrPluginNotFound` instead of exposing the error type itself. See https://dave.cheney.net/2014/12/24/inspecting-errors

Reply:
I think I can reuse `client.IsNotFound` perhaps?

Reply:
if your error type has a func NotFound(){} errdefs.ErrNotFound interface